src/scss/elements/_card.scss

```diff
@@ -0,0 +1,65 @@
+.card{
+  font-family: $main-font;
+  display: grid;
+  max-width: 1250px;
```

Comment:
Why do you have max-width? According to the requirements, it should be full-width all the time.